### PR TITLE
[Merged by Bors] - chore: fix two module docstrings

### PR DESCRIPTION
**Pull request by kbuzzard**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/DFinsupp/Basic.lean [hunk 1]
```diff
@@ -17,7 +17,7 @@ import Mathlib.Order.ConditionallyCompleteLattice.Basic
 /-!
 # Dependent functions with finite support
 
-For a non-dependent version see `data/finsupp.lean`.
+For a non-dependent version see `Data.Finsupp.Defs`.
```
Comment (YaelDillies) [inline Mathlib/Data/DFinsupp/Basic.lean:20]: I've gathered we do
```suggestion
For a non-dependent version see `Mathlib.Data.Finsupp.Defs`.
```
now

File: Mathlib/LinearAlgebra/DFinsupp.lean [hunk 2]
```diff
@@ -11,17 +11,17 @@ import Mathlib.LinearAlgebra.LinearIndependent
 # Properties of the module `Π₀ i, M i`
 
 Given an indexed collection of `R`-modules `M i`, the `R`-module structure on `Π₀ i, M i`
-is defined in `Data.DFinsupp`.
+is defined in `Data.DFinsupp.Basic`.
```
Comment (YaelDillies) [inline Mathlib/LinearAlgebra/DFinsupp.lean:14]:
```suggestion
is defined in `Mathlib.Data.DFinsupp.Basic`.
```

**Review by YaelDillies — APPROVED** [review]
I see more than two docstrings, but I assume that's because you specialised in counting apples 😉 

maintainer merge

**Comment by fpvandoorn** [comment]
bors merge

**Comment by kbuzzard** [comment]
Only two were module docstrings though ;-)

**Pull request closed without merging**